packages/api/controllers/grantCycle.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../models');
+
+const createGrantCycle = async (req, res) => {
+  try {
+    const grant = await db.GrantCycle.create({ ...req.body });
+    if (grant) {
+      return res.status(201).json(grant);
+    }
+  } catch (error) {
+    return res.status(400).send(error.message);
```

Comment:
I wonder if this will mask server/database errors that aren't validation related. Many of the other endpoints return a 500 here, but we may want to handle different kinds of errors differently here?